aspnetcore/blazor/hosting-models.md

```diff
@@ -86,11 +88,11 @@ Blazor WebAssembly includes support for trimming unused code from .NET Core fram
 
 :::moniker-end
 
-## Hosting model selection
+## Should I use Blazor Server or Blazor WebAssembly (WASM)?
```

Comment:
With the addition of Blazor Hybrid, this guidance isn't going to be binary going forward. However, the choice between Blazor Server and Blazor WebAssembly is more nuanced since both are for web apps, while Blazor Hybrid is for native client apps. We could go with your update here and then deal with Blazor Hybrid when we add that content.